### DISCUSSION TITLE
Docusaurus Support

### DISCUSSION THREAD

**Discussion by rustygreen** [nrwl/nx#20249] [Feature Requests]
With NX 17 the Docusaurus plugin is no longer supported (also the plug-in repo is no longer maintained). Is there any new/recommended approaches to working with Docusaurus in an NX repo?

Docusaurus is such a popular and useful tool, it would be nice if NX officially supported it. Any chance we can get an official Docusaurus plugin?

Example:
📦monorepo
 ┣ 📂apps
 ┃ ┗ 📂app1
 ┃ ┗ 📂app1-docs  <---- Docusaurus App
 ┃ ┗ 📂app2
 ┃ ┗ 📂app2-docs  <---- Docusaurus App
 ┗ 📂libs

**Comment by ottobolyos** [comment]
While app-level docs are nice, I’d also like to have workspace-level docs for microservices that are to be used together.
